save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -504,6 +504,8 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                 }
             }
             updateNotificationMessage = ::showNotification
+            organization = state.organization ?: Organization.stub(-1)
```

Comment:
stub was created for tests purpose -- can we request a real organization here? 
also, such update change all fields don't it?

Reply:
That is just a replacement of `null` in order to ignore null checks or unsafe null casts later